### PR TITLE
change service configuration related data model names to common names

### LINKED ISSUE #2206: change service configuration related data model names  to common names
We plan to reuse the design which we did for the service configuration implementation for other usecases such as service actions. Hence must change the namings to something generic. 

- [x] Rename ServiceConfigurationManage to ServiceChangeManage
- [x] Rename ConfigManageScript to ServiceChangeScript - action
- [x] Rename ServiceConfigurationParameter to ServiceChangeParameter
- [x] Rename configManager to changeHandler
- [x] Field names in OCL for serviceConfigurationManage can be same as before. Only model names will change.

### PR DESCRIPTION
**Pull request by WangLiNaruto**
Fixed #2206

image
image

### CODE REVIEW

**Comment by Alice1319** [comment]
LGTM

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/Ocl.java [hunk 1]
```diff
@@ -106,5 +106,5 @@ public class Ocl implements Serializable {
     private String eula;
 
     @Schema(description = "manage service configuration.")
-    private ServiceConfigurationManage serviceConfigurationManage;
+    private ServiceChangeManage serviceChangeManage;
```
Comment (swaroopar) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/Ocl.java:109]: the field name must still be serviceConfigurationManage

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ServiceChangeManage.java [hunk 2]
```diff
@@ -36,10 +36,10 @@ public class ServiceConfigurationManage implements Serializable {
     @Size(min = 1)
     @UniqueElements
     @Schema(description = "The collection of the configuration manage script.")
-    private List<ConfigManageScript> configManageScripts;
+    private List<ServiceChangeScript> serviceChangeScripts;
```
Comment (swaroopar) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ServiceChangeManage.java:39]: same here. the field name must be same as before. Only type must be changed.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ServiceChangeManage.java [hunk 3]
```diff
@@ -36,10 +36,10 @@ public class ServiceConfigurationManage implements Serializable {
...
 
     @Size(min = 1)
     @UniqueElements
     @Schema(description = "The collection of service configuration parameters.")
-    private List<ServiceConfigurationParameter> configurationParameters;
+    private List<ServiceChangeParameter> serviceChangeParameters;
```
Comment (swaroopar) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ServiceChangeManage.java:44]: same here.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/view/ServiceTemplateDetailVo.java [hunk 4]
```diff
@@ -134,5 +134,5 @@ public class ServiceTemplateDetailVo extends RepresentationModel<ServiceTemplate
     private String eula;
 
     @Schema(description = "manage service configuration.")
-    private ServiceConfigurationManage serviceConfigurationManage;
+    private ServiceChangeManage serviceChangeManage;
```
Comment (swaroopar) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/view/ServiceTemplateDetailVo.java:137]: same here.

**Review by swaroopar — CHANGES_REQUESTED** [review]
only model names must change. but the field names must remain as before. (as mentioned in the last requirement of the userstory)

**Comment by wuzhuanhong** [comment]
LGTM

**Comment by profoundwu** [comment]
LGTM

**Comment by baixinsui** [comment]
LGTM

**Comment by yuanxiaojun987** [comment]
LGTM

**Comment by luoping-12345** [comment]
LGTM

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**